internal/root/root.go

```diff
@@ -4,13 +4,13 @@ import "fmt"
 
 func PromptUser() bool {
 	for {
-		fmt.Print("Type [y/n]: ")
+		fmt.Print("Type [Y/n]: ")
 		var res string
 		if _, err := fmt.Scanf("%s", &res); err != nil {
 			return false
 		}
 		switch res {
-		case "yes", "y":
+		case "Yes", "Y":
```

Comment:
The yes option should not only accept a capital Y, people are used to just press y + enter.
Please could you add also lower case `y` and `yes`? Thanks!

Reply:
Yes indeed, work was done